cmd/podman/common/completion.go

```diff
@@ -1522,6 +1523,29 @@ func AutocompleteCgroupManager(cmd *cobra.Command, args []string, toComplete str
 	return types, cobra.ShellCompDirectiveNoFileComp
 }
 
+// AutocompleteContainersConfModules- Autocomplete containers.conf modules.
+func AutocompleteContainersConfModules(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	dirs, err := config.ModuleDirectories()
+	if err != nil {
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	}
```

Comment:
because the option accepts file paths it may be better DirectiveDefault to allow path completion from the shell